packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -40,22 +40,20 @@ Sentry.init({
   integrations: [new BrowserTracing()],
   tracesSampleRate: 0.025,
   maxValueLength: 0,
+  // https://docs.sentry.io/platforms/javascript/guides/react/configuration/filtering/#filtering-error-events
+  ignoreErrors: [
+    // Ignore events related to failed `eth_gasPrice` calls
+    /eth_gasPrice/i,
+    // Ignore events related to failed `eth_getBalance` calls
+    /eth_getBalance/i,
+    // Ignore events related to failed walletConnect calls
+    /Attempt to connect to relay via/i,
+    // Ignore events about window.propertyX being redefined accross multiple extensions
+    /Cannot redefine property/i,
+    // Ignore WC bug until we can update to the latest version, see FS-677
+    /^WebSocket connection failed for host: wss:\/\/relay.walletconnect.org$/i
+  ],
   beforeSend: (event, hint) => {
-    if (event.message) {
```

Comment:
why did we move it from `ignoreErrors` to `beforeSend`

Reply:
no, it's the opposite way